bib/internal/source/source.go

```diff
@@ -58,6 +63,27 @@ func uefiVendor(root string) (string, error) {
 	return "", fmt.Errorf("cannot find UEFI vendor in %s", bootupdEfiDir)
 }
 
+func readImageCustomization(root string) (*blueprint.Customizations, error) {
```

Comment:
Having a (unit) test for this would be really nice, but I don't want to put this burden on you, happy to look at it (in the morning).

Reply:
added unit test